### DISCUSSION TITLE
Create issues instead of pull requests for certain upgrades?

### DISCUSSION THREAD

**Discussion by DCKcode** [renovatebot/renovate#14244] [Suggest an Idea]
I'm using Renovate with package managers (e.g. npm) where `major` version updates have a very high probability of not being able to be upgraded by Renovate. That can easily be addressed by disabling `major` updates for those packages/managers entirely, but that just disables a nice other valuable thing Renovate does: _reminding_ people that major version updates exist.

Would it be possible to configure Renovate in such a way that instead of opening a pull request for a certain `major` version upgrade, it instead opens up/updates an _issue_? This would allow Renovate to still be able to send useful reminders, without creating version updates that are mostly doomed to fail.

**Comment by viceice** [comment]
Did you checked the dependency dashboard issue? It's the only option which is currently supported. You can use that to approve major versions, so renovate will open the PR only on approval.

**Reply by DCKcode** [reply]
Ah I had not considered using it that way! That may be a nice intermediate solution.

It's not _quite_ what I'm looking for though. Having Renovate create issues instead of pull requests comes with some other considerations:

* Having Renovate open issues would allow me to automatically add labels and assignees to the issue to support other processes, like you can do with pull requests. The dependency dashboard wouldn't allow this, as far as I understand.
* Requiring approval for major upgrades may reduce noise of failing MRs, but for many major updates there'd actually be no point to Renovate creating a pull request at all. We'd have to close it and create a manual migration anyway. As a result, there's no point in actually approving that item on the dependency dashboard.

I think that if Renovate could open issues instead of MRs in some scenarios, it could tie better into certain ways of managing issues/PRs, making automation there even better 🙂

**Reply by viceice** [reply]
We use the approval workflow ourselves. If the ci fails because the major update needs manual intervention, we simply push the required changes to the renovate branch.
